frontend/src/components/editor/chrome/wrapper/app-chrome.tsx

```diff
@@ -72,6 +72,7 @@ export const AppChrome: React.FC<PropsWithChildren> = ({ children }) => {
   const sidebarRef = React.useRef<ImperativePanelHandle>(null);
   const terminalRef = React.useRef<ImperativePanelHandle>(null);
   const { aiPanelTab, setAiPanelTab } = useAiPanelTab();
+  const errorCount = useAtomValue(cellErrorCount);
```

Comment:
not sure if its possible to push this down, but errors will create re-renders here (but they are really cheap)